src/day8/re_frame_10x/components/cljs_devtools.cljs

```diff
@@ -218,7 +74,7 @@
 ;; TODO: If we expose ambiance and/or syntax color scheme as settings will need to fix this, maybe by recalculating
 ;; at the time the setting is changed/loaded.
 (def custom-config
-  (merge default-config (base-config :bright :cljs-devtools) #_bright-ambiance-config))
+  (merge default-config (base-config ) #_bright-ambiance-config))
```

Comment:
swap around order of merge so default-config is last i.e.
`(merge (base-config) default-config)`

this is so in theory users can override anything of what is left in our config by modifying default-config

Reply:
I have realized that `default-config` may contain items that `base-config` is trying to customize. For example, `base-config` hides the index tags in lists in reframe-10x while `base-config` styles the index tags. After swapping around the order, the index tags in re-frame-10x will appear instead of being hidden due to the styling overwrite by `default-config`

Reply:
Your right, in that case we have to leave the original merge order @MawiraIke